modules/distr-tracing-deploy-otel-collector.adoc

```diff
@@ -0,0 +1,127 @@
+////
+This module included in the following assemblies:
+- distr_tracing_install/distr-tracing-deploying.adoc
+////
+
+[id="distr-tracing-deploy-otel-collector_{context}"]
+= Deploying distributed tracing data collection
+
+The custom resource definition (CRD) defines the configuration used when you deploy an instance of {OTELName}.
+
+.Prerequisites
+
+* The {OTELName} Operator has been installed.
+//* You have reviewed the instructions for how to customize the deployment.
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Log in to the {product-title} web console as a user with the `cluster-admin` role.
+
+. Create a new project, for example `tracing-system`.
++
+[NOTE]
+====
+If you are installing as part of Service Mesh, the {DTShortName} resources must be installed in the same namespace as the `ServiceMeshControlPlane` resource, for example `istio-system`.
+====
++
+.. Navigate to *Home* -> *Projects*.
+
+.. Click *Create Project*.
+
+.. Enter `tracing-system` in the *Name* field.
+
+.. Click *Create*.
+
+. Navigate to *Operators* -> *Installed Operators*.
+
+. If necessary, select `tracing-system` from the *Project* menu. You may have to wait a few moments for the Operators to be copied to the new project.
+
+. Click the {OTELName} Operator. On the *Details* tab, under *Provided APIs*, the Operator provides a single link.
+
+. Under *OpenTelemetryCollector*, click *Create Instance*.
+
+. On the *Create OpenTelemetry Collector* page, to install using the defaults, click *Create* to create the {OTELShortName} instance.
+
+. On the *OpenTelemetryCollectors* page, click the name of the {OTELShortName} instance, for example, `opentelemetrycollector-sample`.
+
+. On the *Details* page, click the *Resources* tab. Wait until the pod has a status of "Running" before continuing.
+
+[id="distr-tracing-deploy-otel-collector-cli_{context}"]
+= Deploying {OTELShortName} from the CLI
+
+Follow this procedure to create an instance of {OTELShortName} from the command line.
+
+.Prerequisites
+
+* The {OTELName} Operator has been installed and verified.
++
+//* You have reviewed the instructions for how to customize the deployment.
++
+* You have access to the OpenShift CLI (`oc`) that matches your {product-title} version.
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Log in to the {product-title} CLI as a user with the `cluster-admin` role.
```

Comment:
We usually call it simply the "OpenShift CLI" instead of using the {product-title} in front